cycamore/build.sh

```diff
@@ -1,18 +1,30 @@
 #!/bin/bash
 
-mkdir build; cd build;
+mkdir build; 
+cd build;
 export LD_LIBRARY_PATH=$PREFIX/lib/
 export CMAKE_LIBRARY_PATH=$PREFIX/lib/
 export PATH=$PREFIX/bin:$PATH
 export MACOSX_DEPLOYMENT_TARGET=
+
+# # hack fix for hdf5 issues
+# if [[  `uname` == 'Linux' ]]; then
+#   ln -s $PREFIX/lib/libhdf5.so.9 $PREFIX/lib/libhdf5.so.8
+#   ln -s $PREFIX/lib/libhdf5_hl.so.9 $PREFIX/lib/libhdf5_hl.so.8
+# else
+#   ln -s $PREFIX/lib/libhdf5.9.dylib $PREFIX/lib/libhdf5.8.dylib
+#   ln -s $PREFIX/lib/libhdf5_hl.9.dylib $PREFIX/lib/libhdf5_hl.8.dylib
+# fi
```

Comment:
email didn't reply quite right --
I explicitly left comments for whomever comes next with the hdf5 issue. If
you really want them gone, I can remove them, but I think the possible
future help more than makes up for any detraction.